src/pages/api/assistant/trip.ts

```diff
@@ -45,12 +23,13 @@ export default handlerWithAssistantClient<TripApiReturnType>({
     return tryResult(req, res, async () => {
       return ok(
         await client.trip({
-          from,
-          to,
-          searchTime,
-          transportModes:
-            getAllTransportModesFromFilterOptions(transportModeFilter),
-          cursor: tripQuery.cursor,
+          from: tripQuery.from!,
```

Comment:
Any alternatives to using `!` here?

Reply:
Typescript issue more than anything. The check implicitly sets that it cannot be null in this case, but TS doesn't get that. Could do some type guarding. But I'm not entirely happy about the fromId, fromLat nullable etc types the way they are either. But It will require some more refactoring to avoid that now